LocalStorage/Models/LocalStorageCity.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  LocalStorageCity.swift
+//  WeatherDemoApp
+//
+//  Created by ahmed on 06/04/2024.
+//
+
+import Foundation
+
+struct LocalStorageCity {
```

Comment:
```suggestion
struct City {
```

This is a business model, and isn't Codable, so I guess the name should represent an entity instead.